tests/tests.js

```diff
@@ -1442,19 +1453,25 @@ describe('UPS Conversions', () => {
   describe('LLtoUTMUPS', () => {
     describe('convert to UTM when necessary', () => {
       it('80S', ()=> {
-        const utm = converter.LLtoUTMUPS(-80, 0)
-        const expected = "31 441"
-        chai.expect(utm.substr(0, expected.length)).to.equal(expected)
+        const utm = converter.LLtoUTMUPS(-80, 0).split(" ")
+        const expected = ["31", "441", "111"]
+        expected.forEach((_, i) => {
+          chai.expect(utm[i].substr(0, expected[i].length)).to.equal(expected[i])
```

Comment:
The `substr` calls here discard proper checks for values that have the prefix of the expected numbers, e.g., if the UTM returned was "`31C 4415mE 1116mN`". Would a regex match work? The above could be matched against `/31[A-Z] 441mE 111mN/`.